### PR TITLE
{tools}[foss/2018a] git 2.16.1

### PR DESCRIPTION
**Pull request by RvDijk**

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in this PR)
dell.lan - Linux fedora 27, Intel(R) Core(TM) i7-4600U CPU @ 2.10GHz, Python 2.7.14
See https://gist.github.com/fdfaec6f344d7c24f8647f7d12618831 for a full test report.

File: easybuild/easyconfigs/p/Perl/Perl-5.26.1-foss-2018a-bare.eb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name = 'Perl'
+version = '5.26.1'
+versionsuffix = '-bare'
+
+homepage = 'http://www.perl.org/'
```
Comment (verdurin) [inline easybuild/easyconfigs/p/Perl/Perl-5.26.1-foss-2018a-bare.eb]: @RvDijk any reason why you've changed the URL from `https`?

Comment (RvDijk) [reply easybuild/easyconfigs/p/Perl/Perl-5.26.1-foss-2018a-bare.eb]: @verdurin probably from an old template, I'll change it to use https

**Comment by verdurin** [comment]
Going in, thanks @RvDijk!

**Pull request merged**